cargo-miri/src/phases.rs

```diff
@@ -181,6 +181,8 @@ pub fn phase_cargo_miri(mut args: impl Iterator<Item = String>) {
     let target_dir = get_target_dir(&metadata);
     cmd.arg("--target-dir").arg(target_dir);
 
+    // eprintln!("Getting miri flags in phase_cargo_miri");
+    cmd.args(get_miriflags_cargo_mini());
```

Comment:
So you are taking the miri flags and passing them to cargo...? How does that make sense?